src/spec-common/commonUtils.ts

```diff
@@ -257,7 +257,9 @@ export function plainExec(defaultCwd: string | undefined): ExecFunction {
 }
 
 export async function plainPtyExec(defaultCwd: string | undefined, loadNativeModule: <T>(moduleName: string) => Promise<T | undefined>): Promise<PtyExecFunction> {
-	const pty = await loadNativeModule<typeof ptyType>('node-pty');
+	const pty = !!process.env.COMPILE_FOR_PKG
```

Comment:
Please add a comment explaining that this env variable will be inlined at build time.

Reply:
Can do :+1: